### PR TITLE
adding property rules to 5.21 additions

### PR DESCRIPTION
**Pull request by phil198**

### CODE REVIEW

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 1]
```diff
@@ -238,6 +238,23 @@ The column is a `STRING` value specifying a replacement function/procedure if th
...
+----
+[source, cypher, role=noheader]
+----
+GRANT TRAVERSE ON GRAPH * FOR (n:Email) WHERE n.classification IS NULL TO regularUsers
+----
+[source, cypher, role=noheader]
+----
+DENY MATCH {*} ON GRAPH * FOR (n) WHERE n.classification <> 'UNCLASSIFIED' TO regularUsers
+----
+| Introduction of link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/property-based-access-control.adoc[property-based access control] for read privileges. The ability to read, traverse and match nodes based on node property values is now supported in Enterprise Edition.
```
Comment (HannesSandberg) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]: I'm not sure if the link is working, it does not work when I use it in the published version from this PR.
https://neo4j-docs-cypher-1046.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility/#cypher-deprecations-additions-removals-5.21

**Review by HannesSandberg — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc (hunk 1 above)
Comment (phil198) [reply modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]: thanks. I needed to remove `.adoc`. Done now.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1046.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1046.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Review by HannesSandberg — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by JPryce-Aklundh — APPROVED** [review]

**Pull request merged**